app/src/main/java/com/matmoongi/MainActivity.kt

```diff
@@ -17,20 +17,28 @@ import androidx.core.content.ContextCompat
 import com.matmoongi.theme.MatmoongiTheme
 import com.matmoongi.viewmodels.FavoritesViewModel
 import com.matmoongi.viewmodels.SearchViewModel
+import com.matmoongi.viewmodels.UserViewModel
 
 @ExperimentalFoundationApi
 @ExperimentalMaterial3Api
 class MainActivity : AppCompatActivity() {
 
     private val searchViewModel: SearchViewModel by viewModels()
     private val favoritesViewModel: FavoritesViewModel by viewModels()
+    private val userViewModel: UserViewModel by viewModels { UserViewModel.Factory }
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
         checkPermission(this)
 
+        userViewModel.userLoginState.observe(
+            this,
+        ) {
```

Comment:
```suggestion
        userViewModel.userLoginState.observe(this) {
```
혹은
```suggestion
       userViewModel.userLoginState.observe(
           this,
           searchViewModel.refreshMyPageItemList
       )
```
이 올바른 kotlin convention 인 것 같습니다.

Reply:
전자가 더 깔끔해 보이는 것 같아서 전자로 하겠습니다. , 를 붙일 생각이 없었는데 들어가 있네요